packages/components/src/modal/index.tsx

```diff
@@ -125,7 +129,6 @@ function UnforwardedModal(
 
 			if ( openModalCount === 0 ) {
 				document.body.classList.remove( bodyOpenClassName );
-				ariaHelper.showApp();
 			}
```

Comment:
I appreciate the separation of concerns here between modalizing/unmodalizing and adding/removing the body classname